motto/agents/retrieval_agent.py

```diff
@@ -44,12 +44,12 @@ def __init__(self, data_source, chunk_token_size, docs_count, data_dir):
 
     def _process_doc(self, file):
         chunks = {}
-        text = DocProcessor.clear_text(file)
-        chunks['texts'] = DocProcessor.get_text_chunks(text, self.chunk_token_size)
-        chunks['embeddings'] = self.embeddings_getter.get_chunks_embeddings(chunks['texts'])
-        length = len(chunks['texts'])
-        if length > 0:
-            chunks['source'] = [{'source': file}] * length
+        if os.path.exists(file):
```

Comment:
What if the file doesn't exist? Does it require `else`?

Reply:
> What if the file doesn't exist? Does it require `else`?

the empty dict chunks will be returned. and we don not empty chunks into collection
```

length = len(chunks['texts']) if ('texts' in chunks) else 0
 if length > 0:
...
```